### PR TITLE
Add D&I Badging to GSoC List

### PR DESCRIPTION
**Pull request by Nebrethar**
Signed-off-by: Matt Snell <msnell@unomaha.edu>

This is a start! Let me know what you think!

### CODE REVIEW

File: GSoC-Ideas.md [hunk 1]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
 * _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
 * _Recommended:_ Experience with ElasticSearch and Django would be convenient, but can be learned during the project.
 * _Mentors:_ @dlumbrer, @sduenas, @valeriocos
+
+## Idea: Build Workflow Process for CHAOSS Diversity & Inclusion Badging
+
+[ Micro-tasks and place for questions ](https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging/issues/5)
+
+The CHAOSS Diversity and Inclusion Badging Program was founded from necessity in the open source community. The program was created to reward projects and events for positive diversity & inclusion practices, while also holding projects and events accountable to improve. The goal of the Diversity & Inclusion Badging Program is to encourage projects and events to obtain D&I badges for reasons of pride, leadership, self-reflection, and self-improvement on issues critical to building the Internet as a social good.
```
Comment (germonprez) [inline GSoC-Ideas.md]: The CHAOSS Diversity and Inclusion Badging Program began as a way to ensure that D&I metrics can have significant impact in the ways that communities function. The program was created to encourage projects and events towards positive diversity & inclusion practices. The goal of the Diversity & Inclusion Badging Program is to encourage projects and events to obtain D&I badges for reasons of pride, leadership, self-reflection, and self-improvement on issues critical to building the inclusive communities.

File: GSoC-Ideas.md [hunk 2]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
...
+
+The primary goals of the Diversity & Inclusion Badging Program include:
+
+ - Establishing an environment where projects and events can engage in the badging program
```
Comment (germonprez) [inline GSoC-Ideas.md]: Establishing an environment where projects and events can engage in the D&I badging program

File: GSoC-Ideas.md [hunk 3]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
...
+ - Aligning the badging program with best practices, guidelines, and recommendations from Diversity & Inclusion efforts on the CHAOSS project
+ - Connecting with existing badging standards of IMS Global including
+ - Building an open peer-review system for the D&I Badging Program based on the review process of the Journal of Open Source Software
```
Comment (germonprez) [inline GSoC-Ideas.md]: Connecting with existing badging standards of IMS Global including building an open peer-review system for the D&I Badging Program.

File: GSoC-Ideas.md [hunk 4]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
...
+
+The aim of this project regards all of these goals, and the work will help the entire Diversity & Inclusion Badging Program. The volunteer for this project will assist in creating a badge submission workflow and peer-review system for the badging process.
+
+* _Difficulty:_ Medium
+* _Requirements:_ Some experience with GitHub. A willingness to meet new members of the open source community.
```
Comment (germonprez) [inline GSoC-Ideas.md]: * _Requirements:_ Extensive experience with GitHub and the GitHub workflow. A willingness to meet new members of the open source community.

File: GSoC-Ideas.md [hunk 5]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
...
+* _Recommended:_ Experience in the open source community would be a positive, but it is not required.
+* _Mentors:_ Matt Snell, Matt Germonprez
```
Comment (germonprez) [inline GSoC-Ideas.md]: @SMotaal are you interested in participating as a potential mentor in this?

File: GSoC-Ideas.md [hunk 6]
```diff
@@ -126,3 +126,23 @@ The aims will require working with Python, Django and ElasticSearch.
 * _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
 * _Recommended:_ Experience with ElasticSearch and Django would be convenient, but can be learned during the project.
 * _Mentors:_ @dlumbrer, @sduenas, @valeriocos
+
+## Idea: Build Workflow Process for CHAOSS Diversity & Inclusion Badging
+
+[ Micro-tasks and place for questions ](https://github.com/Nebrethar/Docs-Diversity-Inclusion-Badging/issues/5)
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: The issue "Micro-tasks and place for questions" should contain a series of small tasks for a potential student to complete to demonstrate required skills and willingness to put in the effort. This is also a great reason to start communicating with the CHAOSS members.

Right now, the issue only duplicates the content here, which is not the goal.

File: GSoC-Ideas.md (hunk 5 above)
Comment (Nebrethar) [reply GSoC-Ideas.md]: I will ping again about this, but we have shared a few emails about mentoring as a team.

File: GSoC-Ideas.md (hunk 6 above)
Comment (Nebrethar) [reply GSoC-Ideas.md]: Sorry about that! The issue is not yet complete. I will work on that now.

File: GSoC-Ideas.md (hunk 5 above)
Comment (SMotaal) [reply GSoC-Ideas.md]: @germonprez yes, I think the idea of mentoring as a team absolutely works.

Comment (SMotaal) [inline GSoC-Ideas.md]:
```suggestion
* _Mentors:_ Matt Snell, Matt Germonprez, Saleh Abdel Motaal
```

**Comment by SMotaal** [comment]
Completely off point…

https://github.com/chaoss/governance/blob/e7b39875e039fd2603df80602570c18a14b69635/GSoC-Ideas.md#L24

Why the spaces `[ …` and `… ]`? Just it was new to me parsing wise and was inclined to appreciate the divergence :)

**Comment by GeorgLink** [comment]
> Why the spaces [ … and … ]? Just it was new to me parsing wise and was inclined to appreciate the divergence :)

No idea. I was wondering the same thing. It is not essential, so I ignored it to focus on the content. Feel free to create a PR if you want to change it.

File: GSoC-Ideas.md (hunk 6 above)
Comment (GeorgLink) [reply GSoC-Ideas.md]: I would like to merge this PR. Could you please continue to define microtasks? The goal is to have some deliverable that we expect from students to judge their readiness to dive into the mentorship.

**Pull request merged**